websites/M/Miruro/presence.ts

```diff
@@ -0,0 +1,118 @@
+const presence = new Presence({
+		clientId: "1280494760418738247",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "https://i.imgur.com/Wwd5S5f.png",
+}
+
+presence.on("UpdateData", async () => {
+	const { pathname, href } = document.location,
+		presenceData: PresenceData = {
+			name: "Miruro",
+			details: "Browsing",
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+			buttons: [
+				{ label: "Watch Now", url: href },
+				{ label: "Join Discord", url: "https://discord.com/invite/pjsdJp96mY" }, // Miruro Discord server
+			],
+		};
+
+	if (pathname.startsWith("/info")) {
+		const infoTitle = document
+				.querySelector<HTMLElement>("h1.sc-eZxvvn.cywezh")
+				?.childNodes[0]?.textContent.trim(),
```

Comment:
Childnodes[0] isn;t the best option here.
```suggestion
		const infoTitle = document
				.querySelector("h1.sc-eZxvvn.cywezh")?.firstChild?.textContent.trim(),
```